inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/WorkloadMetadataDialog.java

```diff
@@ -32,28 +32,28 @@
     private static final long serialVersionUID = 2797336810690526392L;
 
     public WorkloadMetadataDialog(String aID, SourceDocument aDocument,
-                                  List<String> finshedUsers, List<String> inProgressUser)
+                                  List<String> finishedUsers, List<String> inProgressUsers)
     {
         super(aID);
 
-        if (finshedUsers.size() == 0)
+        if (finishedUsers.isEmpty())
         {
-            finshedUsers.add("-");
+            finishedUsers.add("-");
         }
 
-        if (inProgressUser.size() == 0)
+        if (inProgressUsers.isEmpty())
         {
-            inProgressUser.add("-");
+            inProgressUsers.add("-");
         }
 
 
         //TODO more content
         Label documentName = new Label("documentName", "Document name: "
```

Comment:
Why is the "Document name:" here pare of the label value - this should be in the HTML file. The only dynamic text here is the document name. Same for the other labels.